src/platform/packages/private/kbn-grid-layout/grid/grid_panel/grid_panel.test.tsx

```diff
@@ -36,33 +34,4 @@ describe('GridPanel', () => {
     renderGridPanel();
     expect(screen.getByText('Panel Content panel1')).toBeInTheDocument();
   });
-
```

Comment:
I tried to rewrite these tests so they make sense with the current setup, but they wouldn't provide a lot of value. We would firstly have to add a pretty complicated setup inside the tests (all the dom refs we set up on layout and row level) and then we could only check the implementation details of the `gridLayoutStateManager` (how the state changes). This is not the best approach for having maintainable tests, so I prefer to remove them. I'll think how to replace these tests, probably on the `grid_layout` level (we probably have to add a few tests there too, not necesarily related to this change)

Reply:
I wonder if we could ensure that `mockRenderPanelContents` is only called once in this test as well?

Reply:
We have it already implicitly here: https://github.com/elastic/kibana/pull/206941/files#diff-05bced82dc93ddf863f45a32c931462ee2ff4148d197eb9271cd753714964213R82 (I added a more explicit comment), but the problem is that if this function `renderPanelContents` updates outside, we have to run it again. So this test doesn't guarantee that the consumer of the grid won't cause additional render, but at least we'll be correct in our package.

Reply:
I've added an issue to address the tests separately because I don't want to linger on this PR for too long.  https://github.com/elastic/kibana/issues/208200

The reason for it is that I have another, big PR related to the keyboard on top of this one, so the more I work here, the more conflicts I'll have in the future. I'll address the tests soon! ☺️